tests/unit/jobserver/api/test_jobs.py

```diff
@@ -555,6 +565,40 @@ def test_jobapiupdate_post_with_flags(api_rf):
     assert backend.jobrunner_state["mode"]["v"] == "test"
 
 
+def test_jobapiupdate_post_with_job_request_from_interactive(api_rf):
+    workspace = WorkspaceFactory()
+    job_request = JobRequestFactory(workspace=workspace)
+    job = JobFactory(job_request=job_request, status="running")
+    AnalysisRequestFactory(job_request=job_request)
+
+    now = timezone.now()
+
+    data = [
+        {
+            "identifier": job.identifier,
+            "job_request_id": job_request.identifier,
+            "action": "test",
+            "status": "succeeded",
+            "status_code": "",
+            "status_message": "",
+            "created_at": minutes_ago(now, 2),
+            "started_at": minutes_ago(now, 1),
+            "updated_at": now,
+            "completed_at": seconds_ago(now, 30),
+        },
+    ]
+    request = api_rf.post(
+        "/",
+        HTTP_AUTHORIZATION=job_request.backend.auth_token,
+        data=data,
+        format="json",
+    )
+
+    response = JobAPIUpdate.as_view(get_github_api=FakeGitHubAPI)(request)
+
+    assert response.status_code == 200
```

Comment:
Shouldn't we also check that a github issue was created? Or the api at least called?